src/index.html

```diff
@@ -11,6 +11,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- search bar starts here -->
+    <div class="container-bar">
+      <form action="/" method="get" data-qa="big">
+        <label for="big-search">
```

Comment:
take a look at the comment in `.css` file
```suggestion
    <form action="/" method="get" data-qa="big">
      <label for="big-search" class="container-bar">
```